components/Nav.tsx

```diff
@@ -37,6 +43,8 @@ const Nav = () => {
               </a>
             </li>
 
+          {!user ? 
```

Comment:
Since there's both an if and else result, I'd reverse this check to `user` instead of `!user` (and appropriately switch the outputs) so it's more straightforward.

Reply:
Updated code to reflect these suggestions :)